File: back-end/src/main/java/com/huyphan/dtos/OrderPaginationOptionsDto.java
package com.huyphan.dtos;

import java.util.List;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class OrderPaginationOptionsDto extends PaginationOptionsDto {
	/** Search criteria list. */
	private List<SearchCriteriaDto> criteria;
}

File: front-end/src/app/core/services/dtos/order-pagination-options-dto.ts
import { PaginationOptionsDto } from './pagination-options-dto';
import { SearchCriteriaDto } from './search-criteria-dto';

/** Order pagination options DTO. */
export interface OrderPaginationOptionsDto extends PaginationOptionsDto {
  /** Search criteria list. */
  readonly criteria: readonly SearchCriteriaDto<unknown>[];
}
